File: Unit 1/Fibonacci/Fibo.java
public class Fibo{

    static void fibo(int n, int a, int b) 
    {
        if (n > 0)
        {
            System.out.print(a + ", ");
            fibo(n - 1, b, a + b);
        }
    }

    public static void main(String[] args)
    {
        fibo(5, 0, 1);
    }
}

File: Unit 1-Into/Fibonacci/Fibo.cpp
#include <iostream>
using namespace std;

void fibo(int n, int a, int b)
{
    if (n > 0)
    {
        cout << a << ", ";
        fibo(n - 1, b, a + b);
    }
}

int main()
{
    fibo(5, 0, 1);
    return 0;
}
